[2024-11-29T04:10:57.175+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-29T04:10:57.199+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_elt.dbt_snapshot manual__2024-11-29T04:10:30.483014+00:00 [queued]>
[2024-11-29T04:10:57.205+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_elt.dbt_snapshot manual__2024-11-29T04:10:30.483014+00:00 [queued]>
[2024-11-29T04:10:57.206+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-29T04:10:57.216+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2024-11-29 04:10:30.483014+00:00
[2024-11-29T04:10:57.222+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_elt', 'dbt_snapshot', 'manual__2024-11-29T04:10:30.483014+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/data_elt.py', '--cfg-path', '/tmp/tmpafktzrdf']
[2024-11-29T04:10:57.224+0000] {standard_task_runner.py:91} INFO - Job 64: Subtask dbt_snapshot
[2024-11-29T04:10:57.224+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=622) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-29T04:10:57.225+0000] {standard_task_runner.py:63} INFO - Started process 624 to run task
[2024-11-29T04:10:57.257+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_elt.dbt_snapshot manual__2024-11-29T04:10:30.483014+00:00 [running]> on host 065cc2990949
[2024-11-29T04:10:57.319+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_data_elt' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T04:10:30.483014+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-29T04:10:30.483014+00:00'
[2024-11-29T04:10:57.320+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-29T04:10:57.331+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-29T04:10:57.332+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2024-11-29T04:10:57.339+0000] {subprocess.py:86} INFO - Output:
[2024-11-29T04:11:01.647+0000] {subprocess.py:93} INFO - [0m04:11:01  Running with dbt=1.8.7
[2024-11-29T04:11:02.502+0000] {subprocess.py:93} INFO - [0m04:11:02  Registered adapter: snowflake=1.8.4
[2024-11-29T04:11:03.173+0000] {subprocess.py:93} INFO - [0m04:11:03  Found 4 models, 11 data tests, 1 source, 575 macros
[2024-11-29T04:11:03.177+0000] {subprocess.py:93} INFO - [0m04:11:03
[2024-11-29T04:11:03.178+0000] {subprocess.py:93} INFO - [0m04:11:03  Nothing to do. Try checking your model configs and model specification args
[2024-11-29T04:11:03.964+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-29T04:11:03.965+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-29T04:11:03.989+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nba_data_elt, task_id=dbt_snapshot, run_id=manual__2024-11-29T04:10:30.483014+00:00, execution_date=20241129T041030, start_date=20241129T041057, end_date=20241129T041103
[2024-11-29T04:11:04.047+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-29T04:11:04.075+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-29T04:11:04.080+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
